it/mail/chrome/messenger/mime.properties

```diff
@@ -47,7 +47,7 @@
 ## @name MIME_UNABLE_TO_OPEN_TMP_FILE
 ## @loc None
 # LOCALIZATION NOTE (-1001): Do not translate "\n%.200s.\n\n" below. It will display the name of the temporary directory
--1001=Unable to open the temporary file\n%.200s.\n\n\Check your `Temporary Directory' setting and try again.
+-1001=Impossibile aprire il file temporaneo\n%.200s.\n\n\Controlla le tue impostazioni `Cartella temporanea' e riprova.
```

Comment:
CORRECTION NEEDED. 

There is a minor inconsistency in the primes before and after Cartella temporanea (` and '). The same error is found in the English source text.